code/modules/mob/living/carbon/xenomorph/castes/king/castedatum_king.dm

```diff
@@ -171,7 +171,7 @@
 	spit_delay = 1.1 SECONDS
 
 	// *** Pheromones *** //
-	aura_strength = 6
+	aura_strength = 8 //Primordial king.
```

Comment:
As I said above I don't know much about xeno balance, but having this be almost double the next best `aura_strength` feels like it might be a bit much. The highest other than the queen (5) is an ancient praetorian, with `aura_strength = 4.5`.

Reply:
Fair enough.